src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -128,6 +128,8 @@ const TickDisplay = (() => {
                 width          : config.width || (config.minimize ? 394 : null),
                 height         : config.minimize ? 143 : null,
                 backgroundColor: null,
+                // TODO: we pass UNDEFINED as load event callback,
+                // plot() is executed once
```

Comment:
what is this comment?

Reply:
@negarn on this line
```js
events         : { load: plot(config.plot_from, config.plot_to) },
```
we pass the result of `plot(config.plot_from, config.plot_to)`, which is `undefined` in this case,
this is probably not intentional and what was meant is something like this:
```js
events         : { load: () => {plot(config.plot_from, config.plot_to)} },
```

Reply:
I see, then could we fix it here? or it breaks something?